sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CommonCoderTest.java

```diff
@@ -208,6 +214,22 @@ private static Object convertValue(Object value, CommonCoder coderSpec, Coder co
               convertValue(element, coderSpec.getComponents().get(0), elementCoder));
         }
         return convertedElements;
+      case "urn:beam:coders:global_window:0.1":
```

Comment:
I must have missed this prior for `IterableCoder`, but it can be useful to put these case statements fully inside of `{}` (see `interval_window` case above) -- that lets each `case` use a different set of local variables.